src/coreclr/scripts/superpmi_diffs.py

```diff
@@ -105,43 +117,59 @@ def __init__(self, coreclr_args):
         self.spmi_location = os.path.join(self.script_dir, "artifacts", "spmi")
 
         self.log_directory = coreclr_args.log_directory
-        self.platform_name = coreclr_args.platform
+        self.host_os = coreclr_args.host_os
+        self.target_os = coreclr_args.target_os
         self.arch_name = coreclr_args.arch
 
-        self.os_name = "win" if self.platform_name.lower() == "windows" else "unix"
+        self.jit_os_name = "win" if self.target_os == "windows" else "unix"
         self.host_arch_name = "x64" if self.arch_name.endswith("64") else "x86"
-        self.os_name = "universal" if self.arch_name.startswith("arm") else self.os_name
+        self.jit_os_name = "universal" if self.arch_name.startswith("arm") else self.jit_os_name
 
         # Core_Root is where the superpmi tools (superpmi.exe, mcs.exe) are expected to be found.
         # We pass the full path of the JITs to use as arguments.
         self.core_root_dir = self.script_dir
-        
+
         # Assume everything succeeded. If any step fails, it will change this to True.
         self.failed = False
-        
+
         # List of summary MarkDown files
         self.summary_md_files = []
 
+    def determine_jit_name(self):
```

Comment:
This is the 3rd copy of this method in our scripts, other 2 being in `superpmi.py` and `jitrollingbuild.py`. Can we move this to `jitutil.py`?